dxtbx/model/__init__.py

```diff
@@ -396,6 +397,12 @@ def profiles(self):
     ''' Get a list of the unique profile models (includes None). '''
     from libtbx.containers import OrderedSet
     return list(OrderedSet([e.profile for e in self]))
+  
+  def scaling_models(self):
+    ''' Get a list of the unique scaling models (includes None). '''
+    from libtbx.containers import OrderedSet
+    return list(OrderedSet([e.scaling_model for e in self]))
```

Comment:
Whether set or OrderedSet - in any case please drop the list comprehension brackets